src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,41 @@
+.stars {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+
+  &__star-list {
+    padding: 0;
+    margin: 0;
+    display: flex;
+
+    &--stars-4 :nth-child(-n + 4) {
```

Comment:
You need create all variants of stars block(1, 2.3 etc). 
In this case you can using scss loop ( check in scss documentation)